indico/modules/events/registration/templates/display/regform_display.html

```diff
@@ -100,16 +100,17 @@
 
         <div ng-app="nd" ng-controller="AppCtrl">
             <div nd-reg-form
-                conf-id="{{ event.id }}"
-                conf-sections="{{ sections | tojson | forceescape }}"
-                conf-currency="{{ regform.currency }}"
-                user-info="{{ user_data | tojson | forceescape }}"
-                post-url="{{ request.url | tojson | forceescape }}"
-                check-email-url="{{ url_for('.check_email', regform, management=management) | tojson | forceescape }}"
-                management="{{ management | tojson | forceescape }}"
-                is-moderated="{{ moderated | tojson | forceescape }}"
-                notify-default="{{ session.get('registration_notify_user_default', true) | tojson | forceescape }}">
+                 conf-id="{{ event.id }}"
+                 conf-sections="{{ sections | tojson | forceescape }}"
+                 conf-currency="{{ regform.currency }}"
+                 user-info="{{ user_data | tojson | forceescape }}"
+                 post-url="{{ request.url | tojson | forceescape }}"
+                 check-email-url="{{ url_for('.check_email', regform, management=management) | tojson | forceescape }}"
+                 management="{{ management | tojson | forceescape }}"
+                 is-moderated="{{ moderated | tojson | forceescape }}"
+                 notify-default="{{ session.get('registration_notify_user_default', true) | tojson | forceescape }}">
             </div>
         </div>
+        {{ template_hook('additional-form-sections', event=event, regform=regform, management=management) }}
```

Comment:
what about renaming the hook to `after-regform`? it's not specific to additional sections (and just "form" isn't very clear)